CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features ✨ and improvements 🏁
 
+* Clarify layer requirements in documentation examples related to clustered source methods ([#12680](https://github.com/mapbox/mapbox-gl-js/pull/12680))
+
```

Comment:
You've added an entry to a release that has already been out. Just remove the lines — we're updating the changelog as a part of the release usually.

Reply:
The changelog check is failing... I noticed it is skipped in some other PRs.  What do I need to do to get check to pass so this can be merged?

Reply:
@chriswhong the failing `check-changelog` looks for the `<changelog>` entry in the PR body as a part of the launch checklist, if there is no `skip changelog` label e.g.:

```markdown
 - [x] add an entry inside this element for inclusion in the `mapbox-gl-js` changelog: `<changelog>Clarify layer requirements in documentation examples related to clustered source methods</changelog>`
```